driver-core/src/main/com/mongodb/operation/AsyncChangeStreamBatchCursor.java

```diff
@@ -41,6 +43,9 @@
 
     private volatile BsonDocument resumeToken;
     private volatile AsyncAggregateResponseBatchCursor<RawBsonDocument> wrapped;
+    private final AtomicBoolean isClosed = new AtomicBoolean();
```

Comment:
These AtomicBooleans can now be just booleans as access is always locked by a synchronization block.  See:  AbstractSubscription and add a comment to say they are protected by this.

Reply:
Done